src/core/config/platform.h

```diff
@@ -87,6 +87,16 @@
 #define OPENTHREAD_CONFIG_PLATFORM_UDP_ENABLE 0
 #endif
 
+/**
+ * @def OPENTHREAD_CONFIG_PLATFORM_UDP_SEND_RETRY
+ *
+ * Number of times the stack will retry to send udp packets via kernel.
+ *
+ */
+#ifndef OPENTHREAD_CONFIG_PLATFORM_UDP_SEND_RETRY
+#define OPENTHREAD_CONFIG_PLATFORM_UDP_SEND_RETRY 3
```

Comment:
Move to openthread-posix-config.h
```suggestion
#define OPENTHREAD_POSIX_CONFIG_UDP_SEND_RETRY 3
```